src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -148,8 +148,25 @@ def end_title
 
       # Values
 
+      def bidi_supported?
+        return @bidi_supported unless @bidi_supported.nil?
+        di = Yast::UI.GetDisplayInfo()
+        @bidi_supported = (di ? di["TextMode"] : nil) == false
+      end
+
+      def left_to_right(s)
+        # ncurses would get confused and truncates the text instead
+        return s unless bidi_supported?
+
+        # If we used a plain device.name
+        # then it would display as dev/sda1/ in a RTL context.
+        # Unicode BiDi Left-to-right embedding (LRE),
+        # then Pop Directional Formatting (PDF)
+        format("\u{202A}%s\u{202C}", s)
```

Comment:
This is the actual fix. The rest of the PR 1) works around a libyui-ncurses bug 2) adjusts the tests.

Reply:
In the "after" screenshot I see `boot/efi/`. Is that the expected outcome? Whenever a user tries to use that path for anything, there will probably be an error since the system expects `/boot/efi`.

I understand this must be difficult for users of such an RtoL language (Arabic or Hebrew); if your natural reading direction is right-to-left, how is a user supposed to read such a technical path specification that is really still entirely Western-style? They might still start reading that part from the right, only then to realize that this is really left-to-right; but `boot/efi/` seems wrong in both worlds.